docs/api.md

```diff
@@ -1345,6 +1358,22 @@ If there's no element matching `selector`, the method waits until a matching ele
 
 Shortcut for [page.mainFrame().hover(selector[, options])](#framehoverselector-options).
 
+#### page.innerHTML(selector[, options])
+- `selector` <[string]> A selector to search for an element. If there are multiple elements satisfying the selector, the first will be checked. See [working with selectors](#working-with-selectors) for more details.
+- `options` <[Object]>
+  - `timeout` <[number]> Maximum time in milliseconds, defaults to 30 seconds, pass `0` to disable timeout. The default value can be changed by using the [browserContext.setDefaultTimeout(timeout)](#browsercontextsetdefaulttimeouttimeout) or [page.setDefaultTimeout(timeout)](#pagesetdefaulttimeouttimeout) methods.
+- returns: <[Promise]<null|[string]>>
+
+Resolves to the `node.innerHTML`.
```

Comment:
We always resolve to elements, make this `element.innerHTML`? Same in other methods.